autobahn/wamp/uri.py

```diff
@@ -282,29 +283,35 @@ def is_exception(self):
 
 
 @public
-def register(uri):
+def register(uri, **options):
     """
```

Comment:
This is actually the nicest API (better than what I proposed in a) above) - no breakage of existing code, explicit, safe to use. So let's go with this.